docs/quick-starts/framework/angular/_get-user-information.mdx

```diff
@@ -13,10 +15,86 @@ The `buildAngularAuthConfig()` utility will enable `autoUserInfo` and `renewUser
 :::
 
-Logto uses OIDC [scopes and claims conventions](https://openid.net/specs/openid-connect-core-1_0.html#Claims) to define the scopes and claims for retrieving user information from the ID token and OIDC userinfo endpoint. Both of the "scope" and the "claim" are terms from the OAuth 2.0 and OpenID Connect (OIDC) specifications.
+### Display user information
 
-In short, when you request a scope, you will get the corresponding claims in the user information. For example, if you request the `email` scope, you will get the `email` and `email_verified` data of the user.
+The `OidcSecurityService` provides a convenient way to subscribe to the authentication state as well as user information:
 
-By default, Logto SDK will always request three scopes: `openid`, `profile`. And `offline_access`.
+```ts title="app/app.component.ts"
+import { OidcSecurityService } from 'angular-auth-oidc-client';
+import type { UserInfoResponse } from '@logto/js';
 
-<ScopeClaimList />
+export class AppComponent implements OnInit {
+  isAuthenticated = false;
+  userData?: UserInfoResponse;
+  idToken?: string;
+  accessToken?: string;
+
+  constructor(public oidcSecurityService: OidcSecurityService) {}
+
+  ngOnInit() {
+    this.oidcSecurityService
+      .checkAuth()
+      .subscribe(({ isAuthenticated, userData, idToken, accessToken }) => {
+        console.log('app authenticated', isAuthenticated, userData);
+        this.isAuthenticated = isAuthenticated;
+        this.userData = userData;
+        this.idToken = idToken;
+        this.accessToken = accessToken;
+      });
+  }
+
+  // ...other methods
+}
+```
+
+And use it in the template:
+
+```html title="app/app.component.html"
+<button *ngIf="!isAuthenticated" (click)="signIn()">Sign in</button>
+<ng-container *ngIf="isAuthenticated">
+  <pre>{{ userData | json }}</pre>
+  <p>ID token: {{ idToken }}</p>
+  <p>Access token: {{ accessToken }}</p>
+  <!-- ... -->
+  <button (click)="signOut()">Sign out</button>
+</ng-container>
+```
+
+### Request additional claims
+
+<FindUserInfoMissing method="userData" />
+
+<ScopesAndClaimsIntroduction />
+
+To request additional scopes, you can configure the auth provider configs:
+
+```tsx title="app/app.config.ts"
+import { UserScope, buildAngularAuthConfig } from '@logto/js';
+
+export const appConfig: ApplicationConfig = {
+  providers: [
+    provideHttpClient(withFetch()),
+    provideAuth({
+      config: buildAngularAuthConfig({
+        // ...other configs
+        // highlight-start
+        scopes: [
+          UserScope.Email,
+          UserScope.Phone,
+          UserScope.CustomData,
+          UserScope.Identities,
+          UserScope.Organizations,
+        ],
+        // highlight-end
+      }),
+    }),
+    // ...other providers
+  ],
+};
+```
+
+Then you can access the additional claims in the return value of `userData`.
+
```

Comment:
section: user info that requires network request?

Reply:
Updated

<img width="879" alt="image" src="https://github.com/logto-io/docs/assets/12833674/06faf6bb-d35d-420b-af26-153914360444">